src/app/game.ts

```diff
@@ -271,6 +277,8 @@ export class Game {
     console.log('Updating game')
     console.log('game time', this.gameTimeInSeconds)
     console.log('next attak', this.nextAttackCountdownInSeconds)
+    console.log(this.getBuildings()[0])
+    console.log(this.getBuildings()[1])
```

Comment:
I think this console logs are no necessery anymore.

Reply:
I wrote above - I left the logs to be checked, before merging it will delete them